.github/workflows/tests.yml

```diff
@@ -113,6 +113,12 @@ jobs:
           CUSTOM_S3_SECRET_KEY: ${{secrets.CUSTOM_S3_SECRET_KEY}}
           CUSTOM_S3_ENDPOINT: ${{secrets.CUSTOM_S3_ENDPOINT}}
 
+      - name: Upload coverage to codecov
+        uses: codecov/codecov-action@v1
+        with:
+          file: ./coverage.xml
+          fail_ci_if_error: true
```

Comment:
Does codecov merge these for us rather than having two different reports?

Reply:
I guess if coverage goes up above then we've got a merge scenario

Reply:
Yes, codecov merges them.

Reply:
https://docs.codecov.com/docs/merging-reports